src/project-tests/pomodoro-clock-tests.js

```diff
@@ -78,476 +84,718 @@ export default function createPomodoroClockTests() {
+                      'elapsing')
+                    );
                   }
-                , 1500);
+                }, 1500);
 
               }, 1500);
               observer.disconnect();
             }
           });
-
         });
 
       });
 
-      it('12. When I click the element with the id of "break-decrement", the value within ' +
-        'id="break-length" decrements by a value of 1, and I can see the updated value.',
+      reqNum++;
+      it(`${reqNum}. When I click the element with the id of "break-decrement", 
+      the value within id="break-length" decrements by a value of 1, and I can 
+      see the updated value.`,
       function() {
-        clickButtonsById([_break_min, _break_min, _break_min, _break_min]);
-        FCC_Global.assert.strictEqual(document.getElementById("break-length").innerHTML, "1");
+        clickButtonsById([_breakMin, _breakMin, _breakMin, _breakMin]);
+        assert.strictEqual(
+          document.getElementById('break-length').innerHTML,
+          '1'
+        );
         resetTimer();
-        clickButtonsById([_break_min]);
-        FCC_Global.assert.strictEqual(document.getElementById("break-length").innerHTML, "4");
+        clickButtonsById([_breakMin]);
+        assert.strictEqual(
+          document.getElementById('break-length').innerHTML,
+          '4'
+        );
       });
 
-      it('13. When I click the element with the id of "break-increment", the value within id="break-length" increments by a value of 1, and I can see the updated value.', function() {
-        clickButtonsById(Array(4).fill(_break_plus));
-        FCC_Global.assert.strictEqual(document.getElementById("break-length").innerHTML, "9");
+      reqNum++;
+      it(`${reqNum}. When I click the element with the id of "break-increment", 
+      the value within id="break-length" increments by a value of 1, and I can 
+      see the updated value.`, function() {
+        clickButtonsById(Array(4).fill(_breakPlus));
+        assert.strictEqual(
+          document.getElementById('break-length').innerHTML,
+          '9'
+        );
         resetTimer();
-        clickButtonsById([_break_plus]);
-        FCC_Global.assert.strictEqual(document.getElementById("break-length").innerHTML, "6");
+        clickButtonsById([_breakPlus]);
+        assert.strictEqual(
+          document.getElementById('break-length').innerHTML, '6'
+        );
       });
 
-      it('14. When I click the element with the id of "session-decrement", the value within id="session-length" decrements by a value of 1, and I can see the updated value.', function() {
-        clickButtonsById(Array(4).fill(_sesh_min));
-        FCC_Global.assert.strictEqual(document.getElementById("session-length").innerHTML, "21");
+      reqNum++;
+      it(`${reqNum}. When I click the element with the id of 
+      "session-decrement", the value within id="session-length" decrements by a 
+      value of 1, and I can see the updated value.`, function() {
```

Comment:
Please put the `function() {` part on its own line. I think there are other places you should fix this too.